.eslintrc.json

```diff
@@ -22,5 +22,8 @@
     "plugins": [
         "react"
     ],
-    "rules": {}
+    "rules": {
+        "import/prefer-default-export": "off",
+        "react/forbid-prop-types": 0
+    }
 }
```

Comment:
@neelxie This usually requires you to add a new line just curious if you did

Reply:
@jennizalwango yes it does require if it is not a settings file.